server/controllers/finance/reports/debtors/openDebtors.js

```diff
@@ -101,13 +117,21 @@ function build(req, res, next) {
    * Aggregate SQL for totalling all debts up to the present
    */
   const aggregateSql = `
-    SELECT COUNT(DISTINCT(c.entity_uuid)) AS numDebtors, SUM(c.debit_equiv) AS debit,
-     SUM(c.credit_equiv) AS credit, SUM(c.debit_equiv - c.credit_equiv) AS balance
-    FROM (
-      (SELECT entity_uuid, debit_equiv, credit_equiv FROM posting_journal)
-      UNION (SELECT entity_uuid, debit_equiv, credit_equiv FROM general_ledger)
-    ) AS c
-    WHERE entity_uuid IN (SELECT patient.debtor_uuid FROM patient);    
+    SELECT COUNT(a.entity_uuid) AS numDebtors, SUM(a.debit) AS debit, SUM(a.credit) AS credit, SUM(a.balance) AS balance
+    FROM(
+      SELECT c.entity_uuid, SUM(c.debit_equiv) AS debit, SUM(c.credit_equiv) AS credit,
+        SUM(c.debit_equiv - c.credit_equiv) AS balance
+      FROM (
+        (
+          SELECT entity_uuid, debit_equiv, credit_equiv FROM posting_journal)
+        UNION (
```

Comment:
I'm pretty sure this should be a `UNION ALL` - the original implementation definitely has a bug!